src/styles/components/form/fields/_date-picker.scss

```diff
@@ -114,7 +114,7 @@ $darken: rgba(0, 0, 0, 0.2);
   }
 
   &__panels {
-    background-color: var(--color-rich-black);
+    background-color: var(--datepicker-panel-bg);
```

Comment:
How about scoping the panel color to the component, rather than creating another named global color?

Something like:
```CSS
.date-picker {
  --datepicker-panel-bg: var(--input-panel-bg);
  
  &__panels {
    background: var(--datepicker-panel-bg);
  }
}

// global/_default.scsss
--input-panel-bg: var(--color-rich-black);
// ... other --input-* variables
```

Then you could override in architect either by overwriting `--input-panel-bg` or by targetting `.date-picker.panels { --datepicker-panel-bg }`. What do you think?

Reply:
resolved with 852f8c22f6abe6606b6019be73e6df0dac5e0939